week-4-practise/level-2/exercise-1.js

```diff
@@ -7,19 +7,22 @@ drinking some of it, and emptying it.
 
 We made a start on this for you here: 
 */
-
 var bottle = {
-	volume: 0,
-	fill: function(){
-	 	// calling this function should make you bottles volume = 100; 
-	},
-	drink: function(){
-		// calling this function should decrease your bottles volume by 10;
-	},
-	empty: function(){
-		// this function should return true if your bottles volume = 0
-	}
-}
+  volume: 0,
+  fill: function() {
+    return (bottle.volume = 100);

```

Comment:
When calling `fill()` on your `bottle` object what return type would you expect? Would you expect any at all?